commit 1cc4c99dc6ebeb4c6be46fa56e3ec70086f19c49 (refs/changes/99/4499/2)
Author: Chris Burroughs <chris.burroughs@joyent.com>
Date:   2018-08-24T14:24:07+00:00 (1 year, 1 month ago)
    
    joyent/node-sshpk#51 package.json repository does not point to Joyent
    Reviewed by: Alex Wilson <alex.wilson@joyent.com>
    Approved by: Alex Wilson <alex.wilson@joyent.com>

diff --git a/package.json b/package.json
index af11e3b..e085195 100644
--- a/package.json
+++ b/package.json
@@ -8,7 +8,7 @@
   },
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/arekinath/node-sshpk.git"
+    "url": "git+https://github.com/joyent/node-sshpk.git"
   },
   "author": "Joyent, Inc",
   "contributors": [
